check/fixes.frm

```diff
@@ -2668,3 +2668,54 @@ L F = #x;
 .end
 assert compile_error?("Illegal position for #")
 *--#] Issue499_2 :
+*--#[ Issue512_1 :
+#-
+* Sort which fills SmallExtension:
+
+#: SmallSize 1000K
+#: SmallExtension 1200K
+#: SubSmallSize 1000K
+#: SubSmallExtension 1200K
+#: SubTermsInSmall 2M
+
+Symbol x,n;
+CFunction f,g,prf;
+
+Local test = (<f(1)>+...+<f(150)>)*(<g(1)>+...+<g(100)>);
+.sort
+
+Off parallel;
+PolyRatFun prf;
+
+Identify f(x?) = prf(n-x,n+x);
+
+.end
+assert runtime_error?("Please increase SmallExtension setup parameter.")
+*--#] Issue512_1 :
+*--#[ Issue512_2 :
+#-
+
+* Sort which fills SubSmallExtension:
+
+#: SmallSize 1000K
+#: SmallExtension 1200K
+#: TermsInSmall 2M
+#: SubSmallSize 1000K
+#: SubSmallExtension 1200K
+#: SubTermsInSmall 2M
+
+Symbol x,n;
+CFunction f,g,prf;
+
+Local test = 1;
+.sort
+
+PolyRatFun prf;
+Term;
+	Multiply (<f(1)>+...+<f(150)>)*(<g(1)>+...+<g(100)>);
+	Identify f(x?) = prf(n-x,n+x);
+EndTerm;
+
+.end
+assert runtime_error?("Please increase SubSmallExtension setup parameter.")
```

Comment:
Because `Issue512_2` failed on 32-bit containers (`check-i386 (form/tform)`), you also need to add the following code to prevent it from running on 32-bit machines:
```
#require wordsize >= 4
```